modules/EnsEMBL/Web/Object/Gene.pm

```diff
@@ -732,9 +700,6 @@ sub get_homology_matches {
     my $adaptor_call = $self->param('gene_adaptor') || 'get_GeneAdaptor';
     my %homology_list;
 
-    # Convert descriptions into more readable form
-    my %desc_mapping = $self->get_desc_mapping;
-    
```

Comment:
This hash is never used in the code - we get the information from the compara db now